static/js/stream_ui_updates.js

```diff
@@ -44,6 +44,17 @@ exports.update_regular_sub_settings = function (sub) {
     }
 };
 
+exports.update_change_stream_privacy_settings = function (sub) {
+    var stream_privacy_btn = $(".subscription_settings[data-stream-id='" +
+        sub.stream_id + "']").find(".change-stream-privacy");
```

Comment:
Can you comment on the change in selector in your commit message?

Reply:
Removed the changes. There will only one `change-stream-privacy` element in UI, so don't require to select by stream-row.